.gitlab/ci/bucket-upload.yml

```diff
@@ -148,6 +148,21 @@ upload-packs-to-marketplace:
         python3 ./Tests/Marketplace/copy_and_upload_packs.py -a $PACK_ARTIFACTS -e $EXTRACT_FOLDER -pb "$GCS_MARKET_BUCKET" -bb "$GCS_BUILD_BUCKET" -s $GCS_MARKET_KEY -n $CI_PIPELINE_ID -c $CI_COMMIT_BRANCH -pbp "$STORAGE_BASE_PATH"
       fi
     - section_end "Upload Packs To Marketplace Storage"
+    - section_start "Zip Content Packs From GCS"
+    - |
+      if [ -n "${BUCKET_UPLOAD}" ]; then
+        if [[ $CI_COMMIT_BRANCH != master ]] && [[ $CI_COMMIT_BRANCH != 21\.* ]] && [[ $CI_COMMIT_BRANCH != 22\.* ]]; then
```

Comment:
Let's change the conditions to conform to the others - let's switch the conditions to be positive. So you'll need to rearrange the sections accordingly.

Reply:
I changed the conditions to be positive, but what do you mean by rearranging the sections? what sections?